src/web/layouts/StandardLayout.tsx

```diff
@@ -475,6 +475,7 @@ export const StandardLayout = ({
                                 {showMatchStats && <div id="match-stats" />}
 
                                 {showBodyEndSlot && <div id="slot-body-end" />}
+                                <div id="braze" />
```

Comment:
Can we give a more unique id to this element than just `"braze"` e.g. `"braze-message-container"`?

Reply:
Ah good spot @paulbrown1982. This is actually left over from early work where we were rendering a braze message outside of the banner picker flow (which didn't exist at that point). So actually I think this element just needs to be removed.